### ISSUE TITLE
network shared folder error

### ISSUE THREAD

**Issue by kunshouout** [ClickHouse/ClickHouse#28141]
I have a network shared folder mounted on my server, /large_drive. Once I set <path> entry in the /etc/clickhouse-server/config.xml to /large_drive, I can't login with clickhouse-client. When reset the <path> to the default value, I can login. Is it possible to use network shared folder in clickhouse?

**Comment by kunshouout** [comment]
`
<tmp_path>/home/ken/hh1/click_house/tmp/</tmp_path>
<path>/home/ken/hh1/click_house/</path>
`

**Comment by kunshouout** [comment]
OK, after a lot of try-and-error, I fixed the problem. 
Checked out the error log, /var/log/clickhouse-server/clickhouse-server.err.log, it says
_e.displayText() = DB::Exception: There is no RW access to the disk_

So, it's clear the network folder can't be accessed by clickhouse. However, chmod won't work for network folder, because https://askubuntu.com/questions/1104865/change-permissions-of-windows-mounted-folder-from-linux.

Following the instructions, remount the network folder with 777 file-mode and dir-mode, and then restart the clickhouse-server.
Help this helps for other people running into the same issue.
